dotnet-client-libraries/Selenium-JavaScript/Tests/JavaScriptTests.cs

```diff
@@ -0,0 +1,77 @@
+﻿using FluentAssertions;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Chrome;
+using OpenQA.Selenium.Support.UI;
+using SeleniumExtras.WaitHelpers;
+
+namespace Tests;
+public class JavaScriptTests : IClassFixture<WebFactory>, IDisposable
+{
+    private readonly IWebDriver _driver;
+
+    private readonly string _webUrl = "https://localhost:5055";
+
+    public JavaScriptTests(WebFactory factory)
+    {
+        factory.HostUrl = _webUrl;
+        factory.CreateDefaultClient();
+
+        _driver = new ChromeDriver();
+    }
+
+    [Fact]
+    public void WhenHiddenElement_ThenMakeVisibleLiveTest()
+    {
+        _driver.Navigate().GoToUrl(_webUrl + "/Home/Index");
+
+        var jsExecutor = (IJavaScriptExecutor)_driver;
+
+        // Find the hidden element and ensure it's initially not displayed
+        var hiddenElement = _driver.FindElement(By.Id("hidden-element"));
+        hiddenElement.GetCssValue("display").Should().Be("none");
+
+        // Execute JavaScript to make the hidden element visible
+        jsExecutor.ExecuteScript("document.getElementById('hidden-element').style.display='block';");
+
+        // Wait for the JavaScript execution to take effect (optional, depends on context)
+        var wait = new WebDriverWait(_driver, TimeSpan.FromSeconds(5));
+        wait.Until(ExpectedConditions.ElementIsVisible(By.Id("hidden-element")));
+
+        // Re-check the display property of the hidden element
+        hiddenElement.GetCssValue("display").Should().Be("block");
+    }
+
+    [Fact]
+    public void WhenFoundButton_ThenClickLiveTest()
+    {
+        _driver.Navigate().GoToUrl(_webUrl + "/Home/Index");
+
+        var button = _driver.FindElement(By.Id("testButton"));
+
+        var js = (IJavaScriptExecutor)_driver;
```

Comment:
@01vadim10 
I think it will be great if we are consistent in the naming of variables.
`jsExecutor` reads better than just the  `js` in the the other 2 test methods.